src/auxiliary/auxiliary.jl

```diff
@@ -346,6 +346,20 @@ function register_error_hints()
     return nothing
 end
 
+function read_file(file_path::AbstractString, data_type::Type = Float64)
```

Comment:
This is a rather generic name for something very specific you have in mind. Can't you re-use standard library stuff like `readdlm` or so?

Reply:
Yes, that should be replaced.

Reply:
@warisa-r Can you do this and check if it is working by reading in one of the files we use for testing?

Reply:
`readdllm` functions just like read_file. I made this change in my last commit.